resources/options_lookup.tsv

```diff
@@ -1224,388 +1224,267 @@ HVAC System Is Shared	Only Cooling Shared
 		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
 HVAC System Heating Fuel Oil	"Oil Boiler, 90% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.9	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																		
 		ResidentialAirflow	has_hvac_flue=false																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
 HVAC System Heating Fuel Oil	"Oil Boiler, 95% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.95	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize																																			
 		ResidentialAirflow	has_hvac_flue=false																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
 HVAC System Heating Fuel Oil	Oil Boiler	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, forced draft"	afue=0.76	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																		
 		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
 HVAC System Heating Wood	None																																														
 HVAC System Heating Wood	Wood Stove	ResidentialHVACUnitHeater	fuel_type=wood	efficiency=0.6	fan_power=0	airflow_rate=0	capacity=autosize																																								
 		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
 HVAC System Heating Other Fuel	None																																														
 HVAC System Heating None	None																																														
+HVAC Sizing Show Debug Info	No	ResidentialHVACSizing	show_debug_info=false																																												
+HVAC Sizing Show Debug Info	Yes	ResidentialHVACSizing	show_debug_info=true																																												
 HVAC System Cooling	None																																														
 HVAC System Cooling	"AC, SEER 8"	ResidentialHVACCentralAirConditionerSingleSpeed	seer=8.0	eer=7.3	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	crankcase_capacity=0.0	crankcase_temp=55	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	capacity=autosize	dse=NA																															
-		ResidentialAirflow																																													

```

Comment:
@shorowit I removed them because they don't assign any argument values. Perhaps I don't total understand how the ApplyUpgrade measure works. Can you explain?

Reply:
With the original options_lookup file, if you select this option as an upgrade, the ResidentialAirflow measure will get called a second time after the ResidentialHVACCentralAirConditionerSingleSpeed measure is applied. Since no arguments are provided, it will use the same arguments as were specified for the creation of the existing building.

Imagine the upgrade was from a ducted HVAC system to a non-ducted HVAC system -- calling the ResidentialAirflow measure a second time will remove the ducts.